src/Loop.php

```diff
@@ -124,7 +120,8 @@ public static function get()
      */
     public static function stop()
     {
-        self::get()->stop();
+        $driver = self::$driver ?: self::get();
```

Comment:
Not you're microdeoptimizing :-P Assigns to/reads from static props are more expensive!